lang/ja.json

```
@@ -914,7 +914,7 @@
     "rune": {
       "name": "又オブジェルーン",
       "description": "夢世界を繋ぐ又オブジェ",
-      "condition": "これらの場所を到達する:肉迷宮の通路、青い腕の通路、市街地、霧の湖、うつむき部屋、裏きのこ、溶岩山、ネオンボックス、ヘキサゴン（途切れた道）、吐き気、ノイズの迷宮、タワーズ"
+      "condition": "これらの場所に到達する:肉迷宮の通路、青い腕の通路、市街地、霧の湖、うつむき部屋、裏きのこ、溶岩山、ネオンボックス、ヘキサゴン（途切れた道）、吐き気、ノイズの迷宮、タワーズ"
```

```suggestion
      "condition": "これらの場所に到達する：肉迷宮の通路、青い腕の通路、市街地、霧の湖、うつむき部屋、裏きのこ、溶岩山、ネオンボックス、ヘキサゴン（途切れた道）、吐き気、ノイズの迷宮、タワーズ"
```
Should this be full width?

---

It comes from the habit of typing symbols in half-width characters
I think it's fine to leave it at half-width.